plugins/woocommerce/includes/rest-api/Controllers/Version4/Fulfillments/class-wc-rest-fulfillments-v4-controller.php

```
@@ -0,0 +1,767 @@
+			),
+			'possibilities'     => array(
+				'description' => __( 'Ambiguous shipping providers list.', 'woocommerce' ),
+				'type'        => 'array',
+				'required'    => false,
+				'items'       => array(
+					'type' => 'string',
+				),
+			),
+		);
+		return $schema;
+	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'get_schema_for_get_tracking_number_details'. (Unused Code Rules)

(UnusedPrivateMethod)